libs/base/src/lib/components/wallet/StakeValidation.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FC, useState } from 'react'
+import styled from 'styled-components'
+
+const StakeValidationSC = styled.div`
```

Comment:
We don't have this naming convention anywhere else, but we can start adding it if you think it's helpful. I just hover to see types but YMMV

Reply:
I was using this convention in my last company, and I think it was just a force of habit. I will change it. 
We can revisit to use this convention or not separately.